demo.py

```diff
@@ -0,0 +1,107 @@
+import gradio as gr
+import requests
+import base64
+import tempfile
+import json
+import os
+
+
+SERVER_URL = 'http://localhost:8000'
+OUTPUT = "./demo_outputs"
+cloned_speakers = {}
+
+print("Preparing file structure...")
+if not os.path.exists(OUTPUT):
```

Comment:
Better to use `os.makedirs(..., exist_ok=True)`, that way the folder structure is also repaired if it's incomplete.